src/main/java/com/avaya/letscode/Offer.java

```diff
@@ -20,7 +20,7 @@ public default String getOfferName() {
 
     // For Cart level offers return discount applicable for the offer. 
     // This function will be implemented in all derived classes for cart level offers
-    public default double getDiscountPercentage(Cart cart) {
+    public default double getDiscountedValue(Cart cart) {
         // passsing the cart is to help offer to calculate the discount based on cart value and items in the cart
         // do not use cart.getValue() here as it will cause infinite loop
 
```

Comment:
Now that code returns discounted value and not discount percentage return the cart value without any discount by default here.

```suggestion
return cart.getQuantityBeforeOffer();
```

Reply:
Done!